### DISCUSSION TITLE
dedicated window changes apperiance in main window

### DISCUSSION THREAD

**Discussion by anfien** [zen-browser/desktop#2419] [Ideas]
### Captchas

- [ ] I have read the instructions.
- [ ] I have searched existing issues and avoided creating duplicates.
- [ ] I am not filing an enhancement request.

### What happened?

The Sidebar button takes effect of all windows. so if you have another instance of the zed browser open, all instances are affected by this change. i would expect, that only the selected instance would be effected by this "change".

![0zoejQHK2k](https://github.com/user-attachments/assets/c710a267-9d61-41e9-ae50-8f35c7a61c38)

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

current branch

### What platform are you seeing the problem on?

Windows

**Comment by linear** [comment]
<p><a href="https://linear.app/zen-browser/issue/ZEN-1919/dedicated-window-changes-apperiance-in-main-window">ZEN-1919 dedicated window changes apperiance in main window</a></p>

**Comment by salamijason** [comment]
This one leans on the side of feature requests/suggestions so will be moving it there. Thanks for the idea though!

**Comment by JSGRD22** [comment]
Duplicate of #1416
